About.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles/main.css">
+  <title>Said's Portfolio</title>
+</head>
+
+<body>
+  <div class="wrapper">
+
+    <header class="site-header">
+      <img src="images/download.jpeg" alt="logo-icon" width="77" height="77">
+      <nav class="site-navigation">
+        <ul>
+          <li><a href="Home.html">HOME</a></li>
+          <li><a href="About.html">ABOUT</a></li>
+          <li><a href="Contact.html">CONTACT</a></li>
+        </ul>
+
+      </nav>
+      <h1 class="main-title">Said's Portfolio</h1>
+
+    </header>
+  </div>
+  <main>
+    <article class="main">
+      <h2>About</h2>
+      <p>About me I would describe myself as someone who is honest, caring, intelligent, hardworking, and ambitious. I have a great sense of humour. I am an easy going person and don’t get easily disturbed by down’s in my life. I am spiritual person and
+        have a good knowledge of best best healthy food and life. I also enjoy travelling, watching movies, going out for dinner, and having great intellectual conversations! Hobbies Swimming Coding Reading
+      </p>
+      <div class="">
+        <img src="https://theportfolioyas.files.wordpress.com/2014/05/myportfolio.jpg" alt="">
+        <img src="images/bewer.jpg" alt="said-photo" width="233" height="270">
+      </div>
+    </article>
+
+    <section class="hobbies">
+      <h2>Hobbies</h2>
+      <ul>
+        <li> Swimming </li>
+        <li> Coding </li>
+        <li> Reading </li>
+      </ul>
+    </section>
+  </main>
+
+  <footer>
+    <ul>
+      <li><a href="https://github.com/saidiii111">Git-Hub</a> </li>
+      <li><a href="https://codepen.io/">Code-Pen</a> </li>
+      <li><a href="https://developer.mozilla.org/en-US/docs/Glossary/CSS">MDN</a> </li>
+    </ul>
+    <div class="">
```

Comment:
If you're not adding a value to the class attribute, feel free to remove it completely